addon/components/detail.js

```diff
@@ -112,16 +112,17 @@ export default Component.extend(PropTypeMixin, {
   },
 
   @readOnly
-  @computed('allRenderers', 'renderView')
+  @computed('allRenderers', 'renderValue', 'renderView')
   /**
    * Get store
    * @param {Object} renderers - renderer to component mapping
+   * @param {Object} formValue - current form value
    * @param {BunsenView} view - view being rendered
    * @returns {Object} store
    */
-  store (renderers, view) {
+  store (renderers, formValue, view) {
```

Comment:
Can `formValue` be changed to `renderValue` or `value`? Calling it `formValue is a little confusing because it doesn't match up with the property it is observing.